Sources/PackageGraph/PackageGraph+Loading.swift

```diff
@@ -265,7 +265,7 @@ private func createResolvedPackages(
     }
 
     // Gather all module aliases specified for targets in all dependent packages
-    let pkgToAliasesMap = gatherModuleAliases(for: rootManifests.first?.key, with: packagesByIdentity, onError: observabilityScope)
+    let pkgToAliasesMap = gatherModuleAliases(from: packageBuilders, for: rootManifests.first?.key, with: packagesByIdentity, onError: observabilityScope)
```

Comment:
nit: use clearer variable names, pkgToAliasesMap -> packageAliases

Reply:
onError -> observabilityScope

Reply:
question: is `for: rootManifests.first?.key` a problem? what If there are multiple root packages, or is that not implemented yet?